opentreemap/treemap/instance.py

```diff
@@ -371,6 +371,17 @@ def get_udfc_search_fields(self, user):
 
         return data
 
+    def unlocked_udf_models(self):
```

Comment:
ah, totally. This is an oversight, we already had this discussion about `is_editable` vs `is_unlocked`. You were correct and we went with `is_editable`. Will fix.